packages/core/PluginManager.ts

```diff
@@ -551,4 +558,33 @@ export default class PluginManager {
   ): this {
     return this.addElementType('rpc method', creationCallback)
   }
+
+  addToExtensionPoint<T>(
+    extensionPointName: string,
+    callback: (extendee: T) => T,
+  ) {
+    let callbacks = this.extensionPoints.get(extensionPointName)
+    if (!callbacks) {
+      callbacks = []
+      this.extensionPoints.set(extensionPointName, callbacks)
+    }
+    callbacks.push(callback)
+  }
+
+  evaluateExtensionPoint(extensionPointName: string, extendee: unknown) {
+    const callbacks = this.extensionPoints.get(extensionPointName)
+    let accumulator = extendee
+    if (callbacks) {
+      for (const callback of callbacks) {
+        try {
+          accumulator = callback(accumulator)
+        } catch (error) {
+          console.error(
```

Comment:
can you log the actual error here e.g. console.error(error)

Reply:
Yerp